ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveMaterializedViewsRegistry.java

```diff
@@ -167,12 +168,10 @@ private Loader(Hive db) {
 
     @Override
     public void run() {
-      SessionState ss = new SessionState(db.getConf());
-      ss.setIsHiveServerQuery(true); // All is served from HS2, we do not need e.g. Tez sessions
-      SessionState.start(ss);
       PerfLogger perfLogger = SessionState.getPerfLogger();
-      perfLogger.perfLogBegin(CLASS_NAME, PerfLogger.MATERIALIZED_VIEWS_REGISTRY_REFRESH);
       try {
+        DriverUtils.setUpSessionState(db.getConf());
```

Comment:
nit: This refactor doesn't change the existing behavior. However the method `setUpSessionState` hides that we not just setup the SessionState but also start it. IMHO the original code emphasised this better.

A method name like `setUpAndStartSessionState` would also be better. 
WDYT?

Reply:
@Indhumathi27, please do not rename the original method with `doStart` param, others 2 are fine